modules/varbase_features/varbase_media/varbase_media.install

```diff
@@ -60,4 +60,9 @@ function varbase_media_update_8001() {
   $field_preview_view_mode_config_content_data = (array) Yaml::parse($field_preview_view_mode_config_content);
   $field_preview_view_mode_config->setData($field_preview_view_mode_config_content_data)->save();
 
+	// Change name of Media embed button to be “Media Library".
```

Comment:
@yasmeensalah please change your editor's settings to use double spaces for tabs. No tabs allowed in Drupal coding standards :)